pkg/scheduler/apis/config/types.go

```diff
@@ -144,6 +144,9 @@ type KubeSchedulerProfile struct {
 // Enabled plugins are called in the order specified here, after default plugins. If they need to
 // be invoked before default plugins, default plugins must be disabled and re-enabled here in desired order.
 type Plugins struct {
+	// Requeue is a list of plugins that should be invoked before Pods gets moved from unschedulableQ to backoffQ or activeQ.
+	Requeue PluginSet
```

Comment:
> Does it make sense to have a plugin that implements Requeue, but nothing else?

It doesn't make sense since we only call Requeue which the unsched plugin has. (= if plugin only has Requeue, it will never be called.)

> why would someone want to disable Requeue?

I cannot think of the usecase to disable Requeue since it should benefit everyone.

But, if we want to have it in existing extension point like `PreFilterExtensions`, which extension point would be the best place, do you think?
`Requeue` is mostly for Filter because Filter is the extension point which causes the Pod moved back to the queue the most. But, some few plugins will want to use it with PreEnqueue (schedulniggate plugin), and Reserve (dynamicresource plugin). + we may get more in the future. 
So, I'm not sure if it's really OK to make `Requeue` coupled with specific extension point.

Reply:
It doesn't even have to be coupled with a specific endpoint. It can be an optional interface that any plugin can implement, but it doesn't have to be exposed through the API.